src/lib.rs

```diff
@@ -164,7 +173,10 @@ pub fn run_tests(mut config: Config) -> Result<()> {
 pub fn run_file(mut config: Config, path: &Path) -> Result<std::process::ExitStatus> {
     config.build_dependencies_and_link_them()?;
 
-    let comments = Comments::default();
+    let comments = match std::fs::read_to_string(path) {
+        Ok(content) => Comments::parse(path, &content)?,
+        Err(_) => Comments::default(),
```

Comment:
This seems bad, if there is a single non-utf8-character we just entirely ignore all comments in that file?

Reply:
I've decided to just error if a special comment has non-utf8 before the end of line